pkg/latency/edge/derpmap/derp_regions.go

```diff
@@ -0,0 +1,37 @@
+package derpmap
+
+// maps the DERP "RegionName" to the nearest AWS region name.
+// ref. https://aws.amazon.com/about-aws/global-infrastructure/regions_az/
+var derpRegionNameToAWSRegion = map[string]string{
+	"Amsterdam":     "eu-west-1",
+	"Bangalore":     "ap-south-1",
+	"Chicago":       "us-east-2",
+	"Dallas":        "us-west-2",
+	"Denver":        "us-west-1",
+	"Dubai":         "me-central-1",
+	"Frankfurt":     "eu-central-1",
+	"Hong Kong":     "ap-east-1",
+	"Honolulu":      "us-west-2",
```

Comment:
us-west-3??

Reply:
No, AWS zone or region in Hawaii :0

Picked the one closest (Oregon).

Reply:
let us call it us-west-3... it is just way to far from us-west-2

Reply:
Renamed to us-west-3, and region code, since it's not 1:1 mapped to AWS regions any more. Please let me know if you have any other feedback.